sbc_compassion/models/correspondence_template.py

```diff
@@ -107,6 +107,9 @@ class CorrespondenceTemplate(models.Model):
     checkbox_ids = fields.One2many(
         'sponsorship.correspondence.lang.checkbox', 'template_id',
         default=lambda self: self._get_default_checkboxes())
+    nber_keypoints = fields.Integer(
+        "Number of key points", compute="_compute_template_keypoints",
+        store=True)
```

Comment:
I added a new parameter for the template in order to know how much keypoints are needed during the detection